helmfile/charts/prometheus-alerts/templates/alerts/kubernetes-system-kubelet.yaml

```diff
@@ -164,12 +164,48 @@ spec:
 {{- if .Values.defaultRules.additionalRuleLabels }}
 {{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
 {{- end }}
-    - alert: KubeletDown
+    - alert: KubeletDownWarning
       annotations:
         description: Kubelet has disappeared from Prometheus target discovery.
         runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-kubeletdown
         summary: Target disappeared from Prometheus target discovery.
-      expr: absent(up{job="kubelet", metrics_path="/metrics"} == 1)
+      expr: up{job="kubelet", metrics_path="/metrics"} == 0
+      for: 5m
+      labels:
+        severity: warning
+{{- if .Values.defaultRules.additionalRuleLabels }}
+{{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
+{{- end }}
+    - alert: KubeletDownCritical
+      annotations:
+        description: Kubelet has disappeared from Prometheus target discovery.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-kubeletdown
+        summary: Target disappeared from Prometheus target discovery.
+      expr: up{job="kubelet", metrics_p0ath="/metrics"} == 0
+      for: 15m
+      labels:
+        severity: critical
+{{- if .Values.defaultRules.additionalRuleLabels }}
+{{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
+{{- end }}
+    - alert: LessKubeletsThenNodesWarning
+      annotations:
+        description: There are less kubelets then nodes for the last 5m.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-LessKubeletsThenNodes
+        summary: There are less kubelets then nodes for the last 5m.
+      expr: (sum by (cluster) (kubelet_node_name{cluster=~".*", job="kubelet", metrics_path="/metrics"}) < sum by (cluster)(kube_node_info{cluster=~".*"}))
+      for: 5m
+      labels:
+        severity: warning
+{{- if .Values.defaultRules.additionalRuleLabels }}
+{{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
+{{- end }}
+    - alert: LessKubeletsThenNodesCritical
+      annotations:
+        description: There are less kubelets then nodes for the last 15m.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-LessKubeletsThenNodes
+        summary: There are less kubelets then nodes for the last 15m.
+      expr: (sum by (cluster) (kubelet_node_name{cluster=~".*", job="kubelet", metrics_path="/metrics"}) < sum by (cluster)(kube_node_info{cluster=~".*"}))
```

Comment:
Can't the alerts have the same name but different severity levels?

Reply:
They can, i just wanted to be more clear. But if you prefer to have the same name i can revert it.